xml/selinux.xml

```diff
@@ -176,41 +175,29 @@
     are needed to manage it. The most important part of the work of the
     administrator with regard to &selnx; is managing the policy.
    </para>
-   <warning xml:id="sec-selinux-reference-policy">
-    <title>No default policy included</title>
-    <para>
-     No default or reference policy is provided in
-     &productname;.  &selnx; will not operate without a policy,
-     so you must build and install one.
-      <!-- section removed, it is outdated for SLE 15*. cschroder 2021-7-14 -->
-     <!--The &selnx; Reference Policy Project
-     (<link xlink:href="https://github.com/SELinuxProject/refpolicy/wiki"/>)
-     should be helpful in providing examples and detailed information on creating
-     your own policies, and this chapter also provides guidance on managing
-     your &selnx; policy.-->
-    </para>
-   </warning>
    <para>
     In the &selnx; policy, security labels are applied to different objects
-    on a Linux server. These objects typically are users, ports, processes
+    on a Linux server. These objects typically are users, ports, processes,
     and files. Using these security labels, rules are created that define
-    what is and what is not allowed on a server. Remember, by default
+    what is and what is not allowed on a server. By default,
     &selnx; denies everything, and by creating the appropriate rules you can
     allow the access that is strictly necessary. Rules should therefore
-    exist for all programs that you want to use on a system. Alternatively,
-    you should configure parts of a system to run in unconfined mode, which
-    means that specific ports, programs, users, files and directories are
-    not protected by &selnx;. This mode is useful if you only want to use
-    &selnx; to protect some essential services, while you are not
-    specifically worried about other services. To get a really secure
-    system, you should avoid this.
+    exist for all programs that you want to use on a system.
+   </para>
+   <para>
+    Alternatively, you could configure parts of a system to run in 
+    unconfined mode, which means that specific ports, programs, users, files 
+    and directories are not protected by &selnx;. This mode is useful if you 
+    only want to use &selnx; to protect some essential services. However,
+    this is not a best practice as it leaves your system incompletely 
```

Comment:
I am not happy with that "a" in "a best practice" but I don't know whether to suggest "**the** best practice" or no article at all.

Reply:
"However, this leaves your system incompletely protected, and it is better to apply 
    &selnx; to the whole system."